scripts/beck/st/error_injection.py

```diff
@@ -295,62 +335,77 @@ def induce512kbError():
 				if j > 524288:
 					break
 				else:
-					wbfr.seek(offset)
-					wbfr.write(b'\x33\x33\x44\x44\x22\x22\x11\x11')
-					#ReadMetaData(offset)
+					EditMetadata(offset)
+					#wbfr.seek(offset)
+					#wbfr.write(b'\x33\x33\x44\x44\x22\x22\x11\x11')
+					count = count + 1
+	return count
 
 		
 def verifyLengthOfRecord(recordDict):
-	print("***********Record list will be print here************")
+	count = 0
+	read_metadata_file()
+	logger.info("***********Record list will be logger.info here************")
```

Comment:
logger.info is not a "verb"

Reply:
its typical s/print/loggger.info/g mistake. Thanks I will correct this